http/query_test.go

```diff
@@ -610,3 +610,59 @@ func Test_decodeProxyQueryRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestProxyRequestToQueryRequest_Compilers(t *testing.T) {
+	tests := []struct {
+		name   string
+		pr     query.ProxyRequest
+		want   QueryRequest
+	}{
+		{
+			name: "flux compiler copied",
+			pr: query.ProxyRequest{
+				Dialect: &query.NoContentDialect{},
```

Comment:
I did not know we had a `NoContentDialect`... I guess it will execute the query and then return an empty body in the response?

Reply:
I guess too! I was just looking for the the most straight-forward dialect option that would pass, as I wasn't really sure what we should be doing there for testing.